src/fast_layers.jl

```diff
@@ -40,11 +40,25 @@ struct FastDense{F,F2} <: FastLayer
   σ::F
   initial_params::F2
   bias::Bool
+  precache::Bool
+  cs :: NamedTuple

```

Comment:
Use informative variable names. `cache` instead of `cs`.

Also, this is not type stable, instead parameterize it `::C`.